src/jit/morph.cpp

```diff
@@ -11496,7 +11501,16 @@ GenTreePtr Compiler::fgMorphSmpOp(GenTreePtr tree, MorphAddrContext* mac)
                         }
                     }
                 }
-                fgMorphRecognizeBoxNullable(tree);
+
+                __fallthrough;
+
+            case GT_GT:
+
+                if ((oper != GT_GT) || ((tree->gtFlags & GTF_UNSIGNED) != 0))
```

Comment:
I'm not sure I understand this check. 
It appears that we have to exclude an unsigned GT_GT but everything else is OK.